Lib/importlib/metadata.py

```diff
@@ -0,0 +1,271 @@
+import os
+import re
+import abc
+import sys
+import email
+import itertools
+import contextlib
+
+from configparser import ConfigParser
```

Comment:
The import style is inconsistent between pulling in modules and specific classes. I would stick with just importing the modules.

Reply:
I'm going to suggest that stylistic concerns be addressed upstream first and that style be retained in the port.